src/components/ClientCard/ClientCard.jsx

```diff
@@ -0,0 +1,18 @@
+import "./ClientCard.scss";
+import React from "react";
+
+const ClientCard = ({ name, clientImage }) => {
+  return (
+    <div className="client-card">
+      <img
+        className="client-card__img"
+        src={clientImage}
+        alt="client-image"
+      />
+      <h1 className="client-card__name">{name}</h1>
+      <h3 className="client-card__title">Edit Client</h3>
```

Comment:
If we are going by the designs is this actually going to be a h3 or is it likely to be a link?